tests/model/test_statements.py

```diff
@@ -478,3 +478,17 @@ def test_assignment_create_numeric(load_model_for_test, testdata):
     with pytest.raises(AttributeError):
         Assignment('X', 1.0).free_symbols
     assert Assignment.create('X', 1.0).free_symbols
+
+
+def test_multi_dose_comp_order(load_model_for_test, testdata):
+    model = load_model_for_test(testdata / 'nonmem' / 'pheno.mod')
+    model = set_first_order_absorption(model)
+
+    ode = model.statements.ode_system
+    cb = CompartmentalSystemBuilder(ode)
+    cb.set_dose(cb.find_compartment("CENTRAL"), cb.find_compartment("DEPOT").dose)
+    ode = CompartmentalSystem(cb)
+    model = model.replace(
+        statements=model.statements.before_odes + ode + model.statements.after_odes
+    )
```

Comment:
Very small thing but how come this is in the test? The `assert` doesn't use the model object?

Reply:
It only served a purpose when checking the model myself. It has now been removed!